src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -0,0 +1,88 @@
+import { View } from 'marionette';
+import hbs from 'handlebars-inline-precompile';
+
+import 'scss/modules/buttons.scss';
+import 'scss/modules/sidebar.scss';
+
+import { animSidebar } from 'js/anim';
+
+import Droplist from 'js/components/droplist';
+
+import FiltersSidebarTemplate from './filters-sidebar.hbs';
+
+import './filters-sidebar.scss';
+
+const SidebarGroupsDropList = Droplist.extend({
+  popWidth() {
+    return this.getView().$el.outerWidth();
+  },
+  viewOptions: {
+    className: 'button-secondary--compact w-100',
+    template: hbs`{{ name }}`,
+  },
+  picklistOptions: {
+    attr: 'name',
+  },
+});
+
+const GroupsFilterView = View.extend({
+  className: 'flex flex-align-center',
+  template: hbs`
+    <h4 class="sidebar__label">{{ @intl.patients.sidebar.filters.groups.droplistLabel }}</h4>
+    <div class="flex-grow" data-filter-button></div>
+  `,
+  regions: {
+    filterButton: '[data-filter-button]',
+  },
+  onRender() {
+    const groups = this.getGroups();
+    const selected = groups.get(this.getOption('state').groupId || groups.at(0));
+
+    const groupsFilter = new SidebarGroupsDropList({
+      collection: groups,
+      state: { selected },
+    });
+
+    this.listenTo(groupsFilter.getState(), 'change:selected', (state, { id }) => {
+      this.triggerMethod('select:new:group', id);
```

Comment:
maybe just `select:group`

Reply:
Updated to `select:group` in the last push.